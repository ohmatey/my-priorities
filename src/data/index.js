import Immutable from 'seamless-immutable'

export const prioritiesList = Immutable([
  {
    id: 0,
    name: 'Work',
    hours: 9
  },
  {
    id: 1,
    name: 'Yoga',
    hours: 1
  },
  {
    id: 2,
    name: 'Gym',
    hours: 1
  },
  {
    id: 3,
    name: 'Family Time',
    hours: 1
  }
])
