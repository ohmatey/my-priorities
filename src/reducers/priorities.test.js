import prioritiesReducer from './priorities'
import { prioritiesList } from '../data/'

describe('PrioritiesList component', () => {
  it('matches init snapshot', () => {
    const reducer = prioritiesReducer({}, { type: 'nothing' })

    expect(reducer).toMatchSnapshot()
  })

  it('matches snapshot with list of data', () => {
    const reducer = prioritiesReducer(prioritiesList, { type: 'nothing' })

    expect(reducer).toMatchSnapshot()
  })
})
