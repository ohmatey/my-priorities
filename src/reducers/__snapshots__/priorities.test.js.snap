// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrioritiesList component matches init snapshot 1`] = `Object {}`;

exports[`PrioritiesList component matches snapshot with list of data 1`] = `
Array [
  Object {
    "hours": 9,
    "id": 0,
    "name": "Work",
  },
  Object {
    "hours": 1,
    "id": 1,
    "name": "Yoga",
  },
  Object {
    "hours": 1,
    "id": 2,
    "name": "Gym",
  },
  Object {
    "hours": 1,
    "id": 3,
    "name": "Family Time",
  },
]
`;
